gammapy/irf/psf_king.py

```diff
@@ -130,4 +189,56 @@ def evaluate(self, offset=None, energy=None, interp_kwargs=None):
         values : `~astropy.units.Quantity`
             Interpolated value
         """
-        raise NotImplementedError
+        param = dict()
+        energy = Energy(energy)
+        offset = Angle(offset)
+
+        # Find nearest energy value
+        i = np.argmin(np.abs(self.energy - energy))
+        j = np.argmin(np.abs(self.offset - offset))
+
+        # TODO: Use some kind of interpolation to get PSF
+        # parameters for every energy and theta
+
+        # Select correct gauss parameters for given energy and theta
+        sigma = self.sigma[j][i]
+        gamma = self.gamma[j][i]
+
+        param["sigma"] = sigma
+        param["gamma"] = gamma
+        return param
+
+    def to_table_psf(self, theta=None, offset=None, exposure=None):
+        """
+        Convert king PSF ot table PSF.
+
+        Parameters
+        ----------
+        theta : `~astropy.coordinates.Angle`
+            Offset in the field of view. Default theta = 0 deg
+        offset : `~astropy.coordinates.Angle`
+            Offset from PSF center used for evaluating the PSF on a grid.
+            Default offset = [0, 0.005, ..., 1.495, 1.5] deg.
+        exposure : `~astropy.units.Quantity`
+            Energy dependent exposure. Should be in units equivalent to 'cm^2 s'.
+            Default exposure = 1.
+
+        Returns
+        -------
+        tabe_psf : `~gammapy.irf.EnergyDependentTablePSF`
+            Instance of `EnergyDependentTablePSF`.
+        """
+        # self.energy is already the logcenter
+        energies = self.energy
+
+        # Defaults
+        theta = theta or Angle(0, 'deg')
+        offset = offset or Angle(np.arange(0, 1.5, 0.005), 'deg')
+        psf_value = Quantity(np.empty((len(energies), len(offset))), 'deg^-2')
+
+        for i, energy in enumerate(energies):
+            param_king = self.evaluate(energy, theta)
+            psf_value[i] = Quantity(self.evaluate_direct(offset, param_king["gamma"], param_king["sigma"]), 'deg^-2')
+
+        return EnergyDependentTablePSF(energy=energies, offset=offset,
```

Comment:
Could you create a table PSF instance and check that the value in a certain bin is the same as when you evaluate the KingPSF class at that bin?